CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.23.16] - 2020-08-18
+
```

Comment:
I think we should move the changelog from 0.23.15 to here and say that 0.23.15 should not be used.

Reply:
or can we simply delete 0.23.15 from the changelog?

Reply:
Yes good idea since we're deleting from the catalog.